strawberry/types/fields/resolver.py

```diff
@@ -44,70 +161,54 @@ def __call__(self, *args, **kwargs) -> T:
         return self.wrapped_func(*args, **kwargs)
 
     @cached_property
-    def annotations(self) -> Dict[str, object]:
-        """Annotations for the resolver.
+    def signature(self) -> inspect.Signature:
+        return cached_signature(self._unbound_wrapped_func)
```

Comment:
should we keep the cached property here?

Reply:
I think so, otherwise we will hit the LRU cache 3x per resolver